src/service/persons.with.signficant.control.service.ts

```diff
@@ -0,0 +1,28 @@
+import { Session } from "@companieshouse/node-session-handler";
+import { Request } from "express";
+import { makeApiCallWithRetry } from "./retry.handler.service";
+import { createAndLogErrorRequest, logger } from "../utils/logger";
+import { CompanyPersonsWithSignificantControlResource } from "@companieshouse/api-sdk-node/dist/services/company-psc/types";
+
+export const getCompanyPsc = async (
+  req: Request,
+  companyNumber: string,
+): Promise<CompanyPersonsWithSignificantControlResource | undefined> => {
+  const response = await makeApiCallWithRetry(
+    "companyPsc",
+    "getCompanyPsc",
+    req,
+    req.session as Session,
+    companyNumber
```

Comment:
set max number to 100 or some reasonably high value

Reply:
this endpoint doesn't accept any max_page value, the managing officers one does though.
from sdk:
`public async getCompanyPsc (number: string): Promise<Resource<CompanyPersonsWithSignificantControl>> {`